website/content/tutorials/02-embedded-programming.md

```diff
@@ -60,7 +60,7 @@ The RAM will contain garbage after startup. Our variables aren't initialized and
 
 To do this, we first have to tell the compiler to store our pre-initialized data somewhere in flash memory, so we can copy it into RAM at startup. This is done via the linker script:
 
-```ld
+```
```

Comment:
New version of Zine errors on syntax highlighting it doesn't support, so I dropped `ld` (https://github.com/kristoff-it/zine/issues/46)

Reply:
You can use cpp instead of ld and it works well enough.